unit_tests/test_utils_issues.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import pytest
+
+from sdcm.utils.issues import SkipPerIssues
+
+pytestmark = pytest.mark.integration
+
+
+def test_no_existing_issue(events, params):
+    params.scylla_version = '2023.1.3'
+    with events.wait_for_n_events(events.get_events_logger(), count=1, timeout=10):
+        assert not SkipPerIssues(["ABC"], params)
+
+    events_list = []
+    with events.get_raw_events_log().open() as events_file:
+        for line in events_file.readlines():
+            events_list.append(json.loads(line))
+
+    assert events_list[-1]['message'] == "couldn't parse issue: ABC"
+    assert events_list[-1]['severity'] == 'WARNING'
+
+
+def test_open_issues(params):
+    params.scylla_version = '2023.1.3'
+    assert SkipPerIssues(["https://github.com/scylladb/qa-tasks/issues/1613",
+                          "scylladb/qa-tasks#1613", ], params)
+
+
+def test_closed_issues(params):
+    params.scylla_version = '2023.1.3'
+    assert not SkipPerIssues(
+        ["https://github.com/scylladb/qa-tasks/issues/1614",
+         "scylladb/qa-tasks#1614"], params)
+
+
+def test_closed_issues_with_skip_tag(params):
+    params.scylla_version = '2023.1.3'
+    assert SkipPerIssues(
+        ["https://github.com/scylladb/qa-tasks/issues/1615",
+         "scylladb/qa-tasks#1615"], params)
+
```

Comment:
I don't think the risk is that high, in the sense it would skip or not skip a filter (or a nemesis), the price of miss doing it, is not that high.

we have similar mechanism in dtest (without the tags part), and I don't think we ever run into issue with its implementation.

Reply:
We are talking about adding bunch of unit tests that cover more cases.
It's implementation must be simple.
So, argument `it will not be painful` if we have a bug in the place that can be covered by the requested unit tests doesn't sounds reasonable.

Reply:
Added those tests,  they are part of the  integration tests